feasibility-dsf-process/src/main/java/de/medizininformatik_initiative/feasibility_dsf_process/service/ObfuscateEvaluationResult.java

```diff
@@ -35,11 +41,32 @@ protected void doExecute(DelegateExecution execution, Variables variables) {
     }
 
     private MeasureReport obfuscateFeasibilityCount(MeasureReport measureReport) {
-        var obfuscatedFeasibilityCount = obfuscator.obfuscate(measureReport.getGroupFirstRep().
-                getPopulationFirstRep().getCount());
+        var obfuscatedFeasibilityCount = obfuscator.obfuscate(extractInitialPopulation(measureReport).getCount());
+
+        var obfuscatedMeasureReport = new MeasureReport()
+                .setMeasure(measureReport.getMeasure())
+                .setType(measureReport.getType())
+                .setStatus(measureReport.getStatus())
```

Comment:
Please use a fix status of `complete` and check the original report for that status. We don't like to process `pending` or `error` reports here.